cli/azd/extensions/microsoft.azd.extensions/internal/resources/languages/python/main.py

```diff
@@ -0,0 +1,49 @@
+import sys, os
+
+BASE_DIR = os.path.dirname(__file__)
+sys.path.insert(0, BASE_DIR)
+sys.path.insert(0, os.path.join(BASE_DIR, "generated_proto"))
+
+import asyncio
+from azd_client import AzdClient
+from commands.context_command import ContextCommand
+from commands.listen_command import ListenCommand
+from commands.prompt_command import PromptCommand
+import argparse
```

Comment:
Are there more modern CLI based packages that we can consume vs argparse? Look like [Typer](https://github.com/fastapi/typer) might be a good option to explore?